File: bindings/php/examples/basic.php
<?php

declare(strict_types=1);

require dirname(__DIR__) . '/src/Antech.php';

use Antech\Kdf\Antech;
use Antech\Kdf\Config;

$stored = Antech::hash('correct_horse_battery_staple');
assert(Antech::verify('correct_horse_battery_staple', $stored));

$cfg = Config::default();
$cfg->memoryKib = 1024;
$custom = Antech::hashWithConfig('pw', $cfg);
echo 'needs_rehash ', Antech::needsRehash($custom) ? 'true' : 'false', PHP_EOL;
echo $stored, PHP_EOL;

File: bindings/julia/examples/basic.jl
include(joinpath(@__DIR__, "..", "src", "AntechKdf.jl"))
using .AntechKdf

stored = AntechKdf.hash("correct_horse_battery_staple")
@assert AntechKdf.verify("correct_horse_battery_staple", stored)

cfg = AntechKdf.config_default()
cfg = AntechKdf.Config(UInt32(1024), cfg.salt_length, cfg.block_size, cfg.fan_in, cfg.graph, cfg.output_length)
custom = AntechKdf.hash_with_config("pw", cfg)
println("needs_rehash ", AntechKdf.needs_rehash(custom))
println(stored)
